src/arrayMethodSort.js

```diff
@@ -4,8 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareAsStrings = (a, b) => {
+    const stringA = String(a);
+    const StringB = String(b);
```

Comment:
```suggestion
    const stringB = String(b);
```
змінні варто називати з малої літери